packages/remark-parse/lib/tokenizer.js

```diff
@@ -6,27 +6,23 @@
  * @fileoverview Markdown tokenizer.
  */
 
-'use strict';
+export default factory;
```

Comment:
To my knowledge, no. I tried googling again this morning but can't find any info either way. It was my understanding that the export keyword allocation is meant to be immutable... but I could be mistaken.